### ISSUE TITLE
Provider-direct check-in

### ISSUE THREAD

**Issue by revjtanton** [CodeForBaltimore/Bmore-Responsive#180]
### User Story
As a entity contact
I'd like to be able to check-in directly 
So that I can update my status without speaking with anyone directly

### Acceptance Criteria
- [x] temp user tokens generated for contacts
- [x] contact can access their associated entity via token
- [x] endpoint to trigger email to all contacts with token and link to check-in.

### Notes
This will require work on the front-end as well.

**Comment by msmith125** [comment]
I made myself a contact for three different entities, 2 were Assisted Living Facilities and 1 was a Mixed Housing Facility.  I was Primary Contact for all three and received three separate emails.  The only issue I see is that after I complete the surveys, and try to log-in to HealthCare RollCall, it is like the token has not expired and I cannot log in.  This is via Chrome.  I had to log in via Safari to validate check-ins.  For Chrome, the token may need to be cleared from the cache somehow?
